### ISSUE TITLE
[Add request] Several Adult Domains

### ISSUE THREAD

**Issue by jdk32f2** [blocklistproject/Lists#1037]
URL you wish to be added:

yesporn.vip
cdn.pornhui.com
sexmex.top
yespornvip.com
pornday01.blogspot.com
dash-s1-c10-de-cdn.eporner.com
vid-s1-c10-de-cdn.eporner.com
static-eu-cdn.eporner.com
ads.porndoe.com
fuckfreegirls.com
mylovelydate.life
hitprn.com

UPDATE: more adult domains added here (all confirmed not in the blocklist): https://pastebin.com/ENP2QMVu

Why you believe this should be added: Adult content

Add to list: Adult/Porn

Other info you think we should know:

**Comment by spirillen** [comment]
this is clearly a stolen list, the question is from where and what is the license?

```shell
drill -T ads.porndoe.com
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
.       518400  IN      NS      m.root-servers.net.
com.    172800  IN      NS      a.gtld-servers.net.
com.    172800  IN      NS      b.gtld-servers.net.
com.    172800  IN      NS      c.gtld-servers.net.
com.    172800  IN      NS      d.gtld-servers.net.
com.    172800  IN      NS      e.gtld-servers.net.
com.    172800  IN      NS      f.gtld-servers.net.
com.    172800  IN      NS      g.gtld-servers.net.
com.    172800  IN      NS      h.gtld-servers.net.
com.    172800  IN      NS      i.gtld-servers.net.
com.    172800  IN      NS      j.gtld-servers.net.
com.    172800  IN      NS      k.gtld-servers.net.
com.    172800  IN      NS      l.gtld-servers.net.
com.    172800  IN      NS      m.gtld-servers.net.
porndoe.com.    172800  IN      NS      ns-709.awsdns-24.net.
porndoe.com.    172800  IN      NS      ns-434.awsdns-54.com.
porndoe.com.    172800  IN      NS      ns-1750.awsdns-26.co.uk.
porndoe.com.    172800  IN      NS      ns-1194.awsdns-21.org.
porndoe.com.    86400   IN      SOA     ns-1750.awsdns-26.co.uk. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400
```

**Comment by spirillen** [comment]
OK. here is the complete list and tested, the remaining grey domain will be processed later.

A grey domain, is a domain the bot can't commit with sufficient accuracy.

- `sexmex.co`: https://0xacab.org/my-privacy-dns/matrix/-/issues/152365
- `fullporner.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/122389
- `fullporner.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/197771
- `latestpornvideo.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/512093
- `taboohome.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/229283
- `hdporn92.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/370017
- `hqpornerpro.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/59516
- `porn62.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/23964
- `hqporn.pro`: https://0xacab.org/my-privacy-dns/matrix/-/issues/67658
- `hqpornero.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/604824
- `hqporn.xxx`: https://0xacab.org/my-privacy-dns/matrix/-/issues/27365
- `similar.porn`: https://0xacab.org/my-privacy-dns/matrix/-/issues/49085
- `pornyork.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/96360
- `femdomcult.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/263951
- `hqporner.rocks`: https://0xacab.org/my-privacy-dns/matrix/-/issues/123701
- `pornhd.sex`: https://0xacab.org/my-privacy-dns/matrix/-/issues/45225
- `hqporner.club`: https://0xacab.org/my-privacy-dns/matrix/-/issues/599041
- `hqhole.pro`: https://0xacab.org/my-privacy-dns/matrix/-/issues/189100
- `en.pornohd.blue`: https://0xacab.org/my-privacy-dns/matrix/-/issues/82481
- `pornohd.blue`: https://0xacab.org/my-privacy-dns/matrix/-/issues/82481
- `en8.pornhd8k.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/26099
- `javhub.me`: https://0xacab.org/my-privacy-dns/matrix/-/issues/52941
- `www9.javfun.me`: https://0xacab.org/my-privacy-dns/matrix/-/issues/54115
- `hentaidude.me`: https://0xacab.org/my-privacy-dns/matrix/-/issues/272463
- `pornhd.pet`: https://0xacab.org/my-privacy-dns/matrix/-/issues/67476
- `dogporntube.site`: https://0xacab.org/my-privacy-dns/matrix/-/issues/38322
- `dudethrill.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/416474
- `yoxporn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/66617
- `vip.pornbraze.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/62539
- `xxxtubezoo.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/53395
- `bangyoulater.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/34069
- `tophqporn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/198738
- `trendyxxx.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/202461
- `www.xxxziz.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/127599
- `xxxziz.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/127599
- `wt.pornone.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/22619
- `www.pornonecams.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/451256
- `ptosrd.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/304402
- `pornone.icu`: https://0xacab.org/my-privacy-dns/matrix/-/issues/283665
- `thepornlinks.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/64776
- `www.pornone.asia`: https://0xacab.org/my-privacy-dns/matrix/-/issues/283630
- `pornone.asia`: https://0xacab.org/my-privacy-dns/matrix/-/issues/283630
- `thepornguy.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/30740
- `porneo.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/581461
- `www.porneo.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/581461
- `pornohd.sex`: https://0xacab.org/my-privacy-dns/matrix/-/issues/70537
- `hdpornvideo.xxx`: https://0xacab.org/my-privacy-dns/matrix/-/issues/40893
- `maturehdporn.tv`: https://0xacab.org/my-privacy-dns/matrix/-/issues/61733
- `hd-porn.video`: https://0xacab.org/my-privacy-dns/matrix/-/issues/40596
- `bigtitszone.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/23585
- `xfemaledom.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/152913
- `femdoming.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/342468
- `femdom-pov.me`: https://0xacab.org/my-privacy-dns/matrix/-/issues/125541
- `mixfemdomcc.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/54200
- `onepornlist.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/60741
- `hdhole.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/58063
- `www.slayed.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/28024
- `slayed.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/28024
- `www.deeper.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/24033
- `deeper.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/24033
- `www.milfy.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646250
- `milfy.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646250
- `whoreshub.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/51215
- `thothub.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/1677
- `thothub.to`: https://0xacab.org/my-privacy-dns/matrix/-/issues/64963
- `thothub.is`: https://0xacab.org/my-privacy-dns/matrix/-/issues/50539
- `thothub.mx`: https://0xacab.org/my-privacy-dns/matrix/-/issues/508897
- `thothub.info`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646249
- `thothub.ch`: https://0xacab.org/my-privacy-dns/matrix/-/issues/244805
- `forum.thothub.to`: https://0xacab.org/my-privacy-dns/matrix/-/issues/64963
- `thethothub.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/128792
- `thothub.lol`: https://0xacab.org/my-privacy-dns/matrix/-/issues/129233
- `thothub.vip`: https://0xacab.org/my-privacy-dns/matrix/-/issues/27369
- `thotbook.tv`: https://0xacab.org/my-privacy-dns/matrix/-/issues/128718
- `legitpornsites.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646254
- `onlysearch.co`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646260
- `thotbook.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/208772
- `thotbook.gg`: https://0xacab.org/my-privacy-dns/matrix/-/issues/343939
- `hookupguru.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/22889
- `thots.tv`: https://0xacab.org/my-privacy-dns/matrix/-/issues/554093
- `missingtoofff.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/574973
- `whoreshub.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646266
- `whorehubs.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/23689
- `whoreshub.online`: https://0xacab.org/my-privacy-dns/matrix/-/issues/498911
- `pornwebmasters.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/403447
- `whoreshub.fun`: https://0xacab.org/my-privacy-dns/matrix/-/issues/619393
- `findbestporno.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646239
- `grabporn.me`: https://0xacab.org/my-privacy-dns/matrix/-/issues/587558
- `watchporn.to`: https://0xacab.org/my-privacy-dns/matrix/-/issues/88489
- `myforn.tibet.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/539285
- `goodporn.to`: https://0xacab.org/my-privacy-dns/matrix/-/issues/24170
- `pornspace.me`: https://0xacab.org/my-privacy-dns/matrix/-/issues/555392
- `xxxfree.watch`: https://0xacab.org/my-privacy-dns/matrix/-/issues/67610
- `xmate.best`: https://0xacab.org/my-privacy-dns/matrix/-/issues/420793
- `freeprn.video`: https://0xacab.org/my-privacy-dns/matrix/-/issues/225156
- `tporn.xxx`: https://0xacab.org/my-privacy-dns/matrix/-/issues/31051
- `pornwatch.ws`: https://0xacab.org/my-privacy-dns/matrix/-/issues/62590
- `www.watch-porn.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/51083
- `www.goodporn.to`: https://0xacab.org/my-privacy-dns/matrix/-/issues/24170
- `goodporn.xxx`: https://0xacab.org/my-privacy-dns/matrix/-/issues/40018
- `badassdownloader.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/279011
- `quickporn.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/488050
- `badasshardcore.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/28157
- `badassoftcore.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/28032
- `pornktube.tv`: https://0xacab.org/my-privacy-dns/matrix/-/issues/27159
- `porneznet.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/237112
- `pornez.co`: https://0xacab.org/my-privacy-dns/matrix/-/issues/612972
- `pornez.cam`: https://0xacab.org/my-privacy-dns/matrix/-/issues/544779
- `pornez.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/27417
- `pornez.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/237103
- `pornmz.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/300801
- `paidpornsites.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/461836
- `pornwild.to`: https://0xacab.org/my-privacy-dns/matrix/-/issues/127338
- `youcrazyx.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/27296
- `yrprno.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/327914
- `yourpornflare.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/128029
- `blendporn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/27424
- `daftsex.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/450146
- `severeporn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/127371
- `perverzija.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/30595
- `tube.perverzija.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/30595
- `severeporn.pro`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646267
- `desi-porn.tube`: https://0xacab.org/my-privacy-dns/matrix/-/issues/134426
- `www.ggmansion.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646273
- `manysex.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/366882
- `archivebate.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646252
- `recurbate.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/99028
- `pt.xhamsterlive.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/972
- `onscreens.me`: https://0xacab.org/my-privacy-dns/matrix/-/issues/224083
- `www.onscreens.me`: https://0xacab.org/my-privacy-dns/matrix/-/issues/224083
- `extremeporn.pro`: https://0xacab.org/my-privacy-dns/matrix/-/issues/582194
- `pervertium.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/270631
- `pornve.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/29559
- `lewdninja.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/60816
- `xpornolist.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/65574
- `pornfaze.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/1564
- `pornaddik.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/126944
- `xnhub.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/183713
- `thepornguide.xxx`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646256
- `thotleaks.in`: https://0xacab.org/my-privacy-dns/matrix/-/issues/607688
- `thotleaks.live`: https://0xacab.org/my-privacy-dns/matrix/-/issues/372920
- `thotleaks.tv`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646255
- `leakedthots.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646257
- `cyberleaks.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/133907
- `hardcorensfw.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/58040
- `xpatxpert.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646258
- `leakqueen.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646269
- `masterfap.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/124568
- `myfans.pics`: https://0xacab.org/my-privacy-dns/matrix/-/issues/480936
- `allfansleak.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/322324
- `fapfappy.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/57833
- `leakstorage.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646271
- `porntoc.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/417508
- `thotslife.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/128733
- `of-leaks.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/511323
- `leaksforum.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646262
- `deathpink.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/439396
- `xxxshake.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/24572
- `porndownloadr.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/456851
- `anyxxx.me`: https://0xacab.org/my-privacy-dns/matrix/-/issues/55148
- `milfporn8.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/627924
- `femdomxxx.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646237
- `femdomzzz.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/193737
- `femdomvc.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646247
- `bookmark.xxx`: https://0xacab.org/my-privacy-dns/matrix/-/issues/500269
- `fetishfemdom.adult`: https://0xacab.org/my-privacy-dns/matrix/-/issues/125504
- `femdomup.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/365695
- `ballbusting.cc`: https://0xacab.org/my-privacy-dns/matrix/-/issues/524519
- `sumosear.ch`: https://0xacab.org/my-privacy-dns/matrix/-/issues/128077
- `femdom-joi.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/58231
- `femdorado.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646238
- `simpcity.su`: https://0xacab.org/my-privacy-dns/matrix/-/issues/49145
- `fapello.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/125265
- `gotanynudes.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/54583
- `bitchesgirls.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/123764
- `nudostar.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/61700
- `hornysimp.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/59330
- `fapeza.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/37904
- `lewdstars.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/98819
- `leakhive.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/124572
- `tnudes.to`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646259
- `ttporn.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646264
- `thecamdude.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/64880
- `fapchat.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/288050
- `forum.lewdweb.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/26586
- `cyberslut2069.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/545990
- `her.porn`: https://0xacab.org/my-privacy-dns/matrix/-/issues/60080
- `pornito.xxx`: https://0xacab.org/my-privacy-dns/matrix/-/issues/1492
- `lesbify.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/60692
- `k2s.tv`: https://0xacab.org/my-privacy-dns/matrix/-/issues/269849
- `femdomplanet.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/56728
- `femdomix.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/291565
- `hypnotease.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646265
- `k2s.tube`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646246
- `www.k2s.club`: https://0xacab.org/my-privacy-dns/matrix/-/issues/529016
- `hotk2s.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/559977
- `dominationworld.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/585093
- `k2snetwork.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646244
- `allpornrip.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/55072
- `fetfembdsm.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/375867
- `femdombay.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/216399
- `adultnest.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/55444
- `femdom-fantasy.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/281702
- `k2sxxx.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/378830
- `todominate.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646268
- `k2sporntube.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/205442
- `xvirtualpornbb.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/214370
- `porndoporn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/581234
- `hypnodb.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/454122
- `sididis.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/228770
- `vipxporn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/205605
- `k2shot.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646245
- `ball-busting.cc`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646234
- `femdom888.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646241
- `mysiterip.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646272
- `joi4you.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646263
- `bitetheass.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/524007
- `newfemdomvideos.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/534345
- `cinerotic.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/212527
- `pornevening.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/126872
- `femdomfan.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/216044
- `assoholics.cc`: https://0xacab.org/my-privacy-dns/matrix/-/issues/56221
- `amateurshouse.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646236
- `maxcuckold.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/61090
- `thefapfap.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/60416
- `femaledomination.vip`: https://0xacab.org/my-privacy-dns/matrix/-/issues/502185
- `pornpeach.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/335328
- `fetish-island.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/125629
- `k2s-porn.club`: https://0xacab.org/my-privacy-dns/matrix/-/issues/491752
- `site-rips.club`: https://0xacab.org/my-privacy-dns/matrix/-/issues/422284
- `fhdjav.xyz`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646248
- `sexycandidasses.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/518192
- `fetish-porn.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/278864
- `extremeporn.cc`: https://0xacab.org/my-privacy-dns/matrix/-/issues/549454
- `tabootales.top`: https://0xacab.org/my-privacy-dns/matrix/-/issues/381335
- `kleverig.eu`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646243
- `gangbangbitch.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646240
- `tubefemdom.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646270
- `xhamster3.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/978
- `hardpole.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/58130
- `www.sexvideos.tel`: https://0xacab.org/my-privacy-dns/matrix/-/issues/24704
- `femdomtube.xxx`: https://0xacab.org/my-privacy-dns/matrix/-/issues/23449
- `momvids.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/126365
- `www.momvids.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/126365
- `www.mistresst.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/602891
- `femdomz.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/402934
- `atube.xxx`: https://0xacab.org/my-privacy-dns/matrix/-/issues/624325
- `vipwank.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/67403
- `latestpornvideo.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/340132
- `erpolink.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/346052
- `lovely-milf.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/244361
- `lovely-teen.me`: https://0xacab.org/my-privacy-dns/matrix/-/issues/333989
- `xvirtualfetishbb.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/350124
- `onlyfetish.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/418913
- `mistress-julia.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/61919
- `realbdsmporn.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/63034
- `b-ass.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/35057
- `ilovescat.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/333139
- `bdsmlon.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/35045
- `femdomss.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/57708
- `jerkplanet.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/213994
- `zenporn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/52834
- `fansly.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/125262
- `xfantazy.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/66895
- `admireme.vip`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646235
- `fanvue.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/601289
- `pornofetish.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/62085
- `badtube.cc`: https://0xacab.org/my-privacy-dns/matrix/-/issues/188581
- `badfetish.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/327562
- `newshemalesvideos.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/200572
- `pornmagnet.org`: https://0xacab.org/my-privacy-dns/matrix/-/issues/436147
- `pornmania.cc`: https://0xacab.org/my-privacy-dns/matrix/-/issues/190345
- `fuckkin.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646242
- `vintage-erotic.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/139438
- `watchporninpublic.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/595919
- `public-sex-porn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646261
- `freepublicporn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/58884
- `onlymuscleporn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/61155
- `pornoted.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/539791
- `bestpornsites.guide`: https://0xacab.org/my-privacy-dns/matrix/-/issues/56313
- `bestpornmenu.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/23319
- `pornsitesnow.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/45951
- `fkbae.to`: https://0xacab.org/my-privacy-dns/matrix/-/issues/125482
- `scharferporno.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/64034
- `incest-porn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/465020
- `milliondollarporn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/438873
- `mrmasturbate.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646253
- `www.papoquente.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/522333
- `nicky.xxx`: https://0xacab.org/my-privacy-dns/matrix/-/issues/44810
- `pornscn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/63220
- `l.clips4sale.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/27352
- `narutoporngame.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/278076
- `porndope.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/560580
- `faphuntr.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/644950
- `hussiepass.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/622027
- `seehimfuck.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/102375
- `girlsrimguys.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/40179
- `nsfw247.biz`: https://0xacab.org/my-privacy-dns/matrix/-/issues/213342
- `fxggxt.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/197930
- `www.bang.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/22534
- `bang.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/22534
- `tour2.pornworld.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/30356
- `thewhiteboxxx.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/65052
- `giantesscity.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646251
- `giantess.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/646274
- `fetishlands.net`: https://0xacab.org/my-privacy-dns/matrix/-/issues/487156
- `yesporn.vip`: https://0xacab.org/my-privacy-dns/matrix/-/issues/350656
- `cdn.pornhui.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/641840
- `sexmex.top`: https://0xacab.org/my-privacy-dns/matrix/-/issues/152936
- `yespornvip.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/66794
- `pornday01.blogspot.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/1237
- `dash-s1-c10-de-cdn.eporner.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/25509
- `vid-s1-c10-de-cdn.eporner.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/25509
- `static-eu-cdn.eporner.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/25509
- `ads.porndoe.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/22513
- `fuckfreegirls.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/641842
- `mylovelydate.life`: https://0xacab.org/my-privacy-dns/matrix/-/issues/641841
- `hitprn.com`: https://0xacab.org/my-privacy-dns/matrix/-/issues/41259

**Comment by spirillen** [comment]
>  I built this list myself, by cross checking whether they are being blocked by the blocklist on the pihole search adlist option, I then searched the web for similar terms to test the blocklist and it turns out many were not being blocked.

Here is one tool you should bookmark https://mypdns.eu.org/matrix/is_listed/ as we do import (NSFW only) from major trustworthy blacklist you will find a rather high accuracy by testing (searching) against our lists

And you can bulk add via our FF browser plugin https://0xacab.org/my-privacy-dns/matrix/-/blob/master/tools/client_addon.md

and if you find a page with only porn url... you can mark them all => right click => report urls as...

**Comment by AxiomOperator** [comment]
This issue is being closed as part of a repository cleanup and issue triage refresh.

We are starting with a clean slate to better manage the blocklist project going forward. If this issue is still relevant, please feel free to open a new issue with:
- Updated information
- Current evidence/examples
- Clear description of the request

Thank you for your contribution to the Block List Project!
